{"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1710", "repository_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT", "labels_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1710/labels{/name}", "comments_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1710/comments", "events_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1710/events", "html_url": "https://github.com/Significant-Gravitas/Auto-GPT/issues/1710", "id": 1669567026, "node_id": "I_kwDOJKSTjM5jg5Iy", "number": 1710, "title": "SYSTEM:  Command google returned: Error: No module named 'google.auth'", "user": {"login": "GeekLabsUK", "id": 68666817, "node_id": "MDQ6VXNlcjY4NjY2ODE3", "avatar_url": "https://avatars.githubusercontent.com/u/68666817?v=4", "gravatar_id": "", "url": "https://api.github.com/users/GeekLabsUK", "html_url": "https://github.com/GeekLabsUK", "followers_url": "https://api.github.com/users/GeekLabsUK/followers", "following_url": "https://api.github.com/users/GeekLabsUK/following{/other_user}", "gists_url": "https://api.github.com/users/GeekLabsUK/gists{/gist_id}", "starred_url": "https://api.github.com/users/GeekLabsUK/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/GeekLabsUK/subscriptions", "organizations_url": "https://api.github.com/users/GeekLabsUK/orgs", "repos_url": "https://api.github.com/users/GeekLabsUK/repos", "events_url": "https://api.github.com/users/GeekLabsUK/events{/privacy}", "received_events_url": "https://api.github.com/users/GeekLabsUK/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 5272676262, "node_id": "LA_kwDOJKSTjM8AAAABOkanpg", "url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/labels/question", "name": "question", "color": "d876e3", "default": true, "description": "Further information is requested"}], "state": "open", "locked": false, "assignee": null, "assignees": [], "milestone": null, "comments": 1, "created_at": "2023-04-15T20:47:08Z", "updated_at": "2023-04-17T12:36:19Z", "closed_at": null, "author_association": "NONE", "active_lock_reason": null, "body": "### Duplicates\n\n- [X] I have searched the existing issues\n\n### Steps to reproduce \ud83d\udd79\n\nauto gpt tries to use google\n\n### Current behavior \ud83d\ude2f\n\nget the error SYSTEM:  Command google returned: Error: No module named 'google.auth'\n\n### Expected behavior \ud83e\udd14\n\n_No response_\n\n### Your prompt \ud83d\udcdd\n\n```yaml\r\n# Paste your prompt here\r\n```\r\n", "reactions": {"url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1710/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/Significant-Gravitas/Auto-GPT/issues/1710/timeline", "performed_via_github_app": null, "state_reason": null}